bundles/org.openhab.core/src/test/java/org/openhab/core/library/types/DateTimeTypeTest.java

```diff
@@ -290,6 +291,21 @@ public void epochTest() {
         assertThat(epochStandard, is(zdtStandard));
     }
 
+    @Test
+    public void relativeTest() {
+        DateTimeType dt1 = new DateTimeType("2019-06-12T17:30:00Z");
+        DateTimeType dt2 = new DateTimeType("2019-06-12T00:00:00+0000");
```

Comment:
To ensure that this checks for the same instant:

dt1 = "2019-06-13T01:00:00+02"
dt2 = "2019-06-12T23:00:00Z"

Reply:
Changed